src/components/Announce/blogpost.js

```diff
@@ -0,0 +1,6 @@
+export default {
```

Comment:
I backdated the post so it doesn't show up at the top and merged it so you can test as if it were real